kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
I'm not sure about why you are getting this error as this never happened to me. Is the ES deployment running before you create the server deployment?